core/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -47,6 +48,8 @@ public static QueryBuilder createQuery(Random r) {
             case 3:
                 return createMultiTermQuery(r);
             case 4:
+                return new GeoDistanceRangeQueryTests().createTestQueryBuilder();
```

Comment:
Well, except the generic. But I think that can be fixed too? You're right, I think its worth exploring in a separate cleanup issue?